src/components/CoinCard/index.tsx

```diff
@@ -0,0 +1,27 @@
+interface IProps {
+  coinInfo: any;
+  price: string;
+}
+
+const CoinCard = ({ coinInfo, price }: IProps) => {
+  return (
+    <div className="bg-gray-700 rounded-md h-20 mx-96 my-16">
+      <div className="w-full flex p-2 items-center">
+        <div className="w-1/6">
```

Comment:
these 3 divs can be removed, you can use one image tag, two divs, without h1 tags.